### ISSUE TITLE
Some feedback on cartography and functionality

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/RIM#6]
Hi @kbartelt. I just downloaded the Inundation project you sent me (with `project_rs.xml` riverscapes project and `inundation.xml` business logic anda bunch of `*.lyr` files all in root of project). Good news! It works! Here is a video with some feedback on how it worked for me, what was logical and some potential fixes. The only layer that didn't work was the 2017 image. Everything else comes in and there are nit-picky comments. @philipbaileynar may get a kick out of this too. 

[![](http://img.youtube.com/vi/HvLN6axmlyk/0.jpg)](http://www.youtube.com/watch?v=HvLN6axmlyk "")

Nice job. Post follow up questions here.

**Comment by kbartelt** [comment]
Great! Thank you for the feedback and suggestions Joe!

One question- 

So the mapped DCE you were looking at with the old 2017 imagery was actually just an estimation of pre- beaver inundation. For this site (and many) there is no imagery predating beaver activity and so I did my best to approximate what pre beaver inundation extent might have been (I think we talked about doing this a while back). I did include this in my fake metadata (see below), but do you have any suggestions for how I could make this more clear in RAVE/ Arcmap? Do you still think the pre beaver inundation extents is something that is OK to approximate? For this particular site I plan to actually map the 2017 inundation extent because it is a good example of the blown-out state (i think there will be some pockets of backwater from remnants of failed dams) but for other sites I was only planning to have the mapped orthophoto inundation extent along with an estimated pre beaver inundation extent.

```
            <Name>Inundation</Name>
            <Path>02_Mapping/DCE_02/inundation.shp</Path>
            <AP ref="none" />
            <MetaData>
                <Meta Name="mapper">Karen Bartelt</Meta>
                <Meta Name="date_mapped">02042020</Meta>
                <Meta Name="field_visit">no</Meta>
                <Meta Name="confidence">low</Meta>
                <Meta Name="notes">This is an estimation of what the pre beaver inundation extent might be based off of historical imagery (mostly AP_02, June 2017 google earth) and inferences made in the field and upstream and downstream of the site. </Meta>
            </MetaData>
```

below is a checklist for myself based on your suggestions
- [ ] update the default collapse/ expand settings on RAVE project explorer
- [ ] rename output to analyses and reorder to be below mapping
- [ ] change BRAT to be the whole HUC8 like VBET is (first need to change the way the BRAT metrics are extracted because I pull Qlow, Q2, and stream power from BRAT)
- [ ] update valley bottom centerline symbology
- [ ] change valley bottom polygon so that the ends aren't cut off by the site boundary 
- [ ] check out symbology for thalwegs from CHaMP topo
     - I am using the same thalweg categories as GUT, so can probably just use GUT symbology
- [ ] dam state symbology - make easier to see difference between breached and blown out. Green for intact is hard to see
- [ ] rename plots and fix color (still need to code this in MatPlotLib)
- [ ] work on symbology for the change detection (also need to code this up still, this one was produced manually)

**Comment by kbartelt** [comment]
@joewheaton I'm glad you brought up the 2017 imagery not loading. This was something I meant to ask about but forgot.

@philipbaileynar I have a lot of imagery that is in .jpg format (google earth photos that I georeferenced in ArcMap). I was not able to figure out how to get these to work properly with RAVE. They show up in the RAVE project explorer but wont load into the ArcMap TOC.

This is how it is coded in the business logic xml:

`<Node xpathlabel="Name" xpath="Inputs/Raster[@id='AP_02']" type="raster" symbology="" />`

and this is how it is coded in the project xml:
```
        <Raster id="AP_02">
            <Name>2017 June</Name>
            <Path>01_Inputs/01_Imagery/AP_02/2017_june_google_earth.jpg</Path>
            <MetaData>
                <Meta name="image_date">2017-06</Meta>
                <Meta name="source">Google Earth</Meta>
                <Meta name="flow_stage">low</Meta>
                <Meta name="ortho_processing_software"></Meta>
                <Meta name="orthomosaic_accuracy"></Meta>
            </MetaData>
        </Raster>
```

**Comment by philipbaileynar** [comment]
I had not anticipated georeferenced JPG rasters. 

I think both the business logic and project file are appropriate. I just need to investigate how to get these into the map. If I struggle, we might need another node type such as "geojpg". But I'm fairly confident that I can get it to work with what you have.

Remind me how JPGs are georeferenced? Is the coordinate system and transformation contained within the JPG header, or is there a sidecar file need to load these into ArcMap and have them appear in the correct place on Earth?

**Comment by kbartelt** [comment]
I believe the coordinate system and transformation is contained in the .jgw file

![image](https://user-images.githubusercontent.com/39836333/82099162-d5530b80-96c3-11ea-9d0e-0a9bbaf0be5b.png)

**Comment by joewheaton** [comment]
Hi @kbartelt sorry I missed this:

> So the mapped DCE you were looking at with the old 2017 imagery was actually just an estimation of pre- beaver inundation. For this site (and many) there is no imagery predating beaver activity and so I did my best to approximate what pre beaver inundation extent might have been (I think we talked about doing this a while back). 

Yes, of course these are estimates. I think you need to look into how positional polygon mapping is sometimes expressed with error estimates or uncertainty. So for positioning of a line are you ±1 m, ±0.5 m, etc. A general rule of thumb is somewhere in the 1-2.5 the pixel resolution. 

For area, what would this be? If you're digitizing 300 m of channel length, and you think you could be ±1 m on either side, you'd be ±600 m^2. Now if the channel is 1 m wide, that's 300 m^2 ± 600 m^2 or ±200%!. However, if it is 5 m wide, that's only that's 1500 m^2 ± 600 m^2 or ±40%!

> I did include this in my fake metadata (see below), but do you have any suggestions for how I could make this more clear in RAVE/ Arcmap?

Include some Metadata estimates of mapping uncertainty like I show above.

 

> Do you still think the pre beaver inundation extents is something that is OK to approximate? 

Absolutely! You can reasonably estimate channel width and even compare it with the Beechie bankfull width estimate empirical formulae. Even with large uncertainty on pre, you could still be looking at significant differences when those are propagated into the changes. Follow the logic through on error propagation we do in GCD. Just apply it to area or to integrated width (length). 

> For this particular site I plan to actually map the 2017 inundation extent because it is a good example of the blown-out state (i think there will be some pockets of backwater from remnants of failed dams) but for other sites I was only planning to have the mapped orthophoto inundation extent along with an estimated pre beaver inundation extent.

I think you need to express some uncertainty in all your mapping estimates. Read some papers on digitizing...  Jack and Peter's old student, Dave Gaueman has a nice over the top example of this: 
- Gaeuman D, Schmidt JC, Wilcock PR. 2003. Evaluation of in-channel gravel storage with morphology-based gravel budgets developed from planimetric data. Journal of Geophysical Research- Earth Surface 108 : 1–16.
[1702_Gaeuman 2003.pdf](https://github.com/Riverscapes/inundation/files/4653696/1702_Gaeuman.2003.pdf)

**Comment by joewheaton** [comment]
@CHafen we'll discuss any clean up items on this one before closing it.

**Comment by CHafen** [comment]
@joewheaton I watched this video and read through this thread. It looks like there are no outputs for the projects that I've looked at which I think is just change detection. One of the to do things @kbartelt has listed earlier in this thread is "work on symbology for the change detection (also need to code this up still, this one was produced manually)." I assume that this has not been coded yet, is that right? Is that something that is going to wait until a later version to get added in?
